net.gwyddion.Gwyddion.json

```diff
@@ -0,0 +1,35 @@
+{
+    "app-id": "net.gwyddion.Gwyddion",
+    "branch": "stable",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "command": "gwyddion",
+    "rename-icon": "gwyddion",
```

Comment:
I would use a copy-icon: true too. Maybe the icon is used somewhere else with the old name.

Reply:
Okay!